docs/kop.md

```diff
@@ -1,5 +1,5 @@
 ---
-dockerfile: "https://github.com/streamnative/kop/releases/download/v2.8.0-rc-202105291235//pulsar-protocol-handler-kafka-2.8.0-rc-202105291235.nar"
+dockerfile: "https://github.com/streamnative/kop/releases/download/v2.8.0.1/pulsar-protocol-handler-kafka-2.8.0.1.nar"
```

Comment:
```suggestion
download: "https://github.com/streamnative/kop/releases/download/v2.8.0.1/pulsar-protocol-handler-kafka-2.8.0.1.nar"
```

Reply:
@tuteng since we don't have docker image for KoP, shall we replace it with "download"?